coderedcms/models/page_models.py

```diff
@@ -1293,7 +1291,11 @@ def send_summary_mail(self, request, form, processed_data):
             message_args['reply_to'] = template_reply_to.render(context).split(',')
 
         # Send email
+        self.__send_mail(request, message_args)
```

Comment:
This message is not HTML, it is plain text. So that should probably be an argument to `send_mail()`. In the current change it is hard-coded to HTML-only.

Reply:
I have added an optional parameter (default is text, is that correct?)